integration-libs/epd-visualization/components/visual-viewer/toolbar/visual-viewer-animation-slider/visual-viewer-animation-slider.service.spec.ts

```diff
@@ -7,10 +7,10 @@ import {
 import { TestBed } from '@angular/core/testing';
 import { LanguageService, TranslationService } from '@spartacus/core';
 import { EventListenerUtils } from '@spartacus/epd-visualization/root';
+import { DirectionMode } from 'projects/storefrontlib/layout/direction/config/direction.model';
+import { DirectionService } from 'projects/storefrontlib/layout/direction/direction.service';
```

Comment:
Why not from from '@spartacus/storefront' ?

Reply:
no reason, to me it's the same as how it is before but instead relatively doing it as it was done before, I made it absolute

Reply:
I can change it